tests/test_api.cpp

```diff
@@ -284,6 +284,42 @@ TEST( ApitTest, LoggerApi )
   EXPECT_EQ( receivedLogMessage, "No driver with index: -1" );
 }
 
+TEST( ApitTest, BuildUrisApi )
+{
+  MDAL_SetLoggerCallback( &_testLoggerCallback );
+  MDAL_SetLogVerbosity( MDAL_LogLevel::Debug );
+
+  std::vector<std::pair<std::string, std::string>> testScenarios
+  {
+    {
+      test_file( "/2dm/regular_grid.2dm" ),
+      "2DM:\"" + test_file( "/2dm/regular_grid.2dm" ) + "\""
+    },
+    {
+      test_file( "/ugrid/1dtest/dflow1d_map.nc" ),
+      "Ugrid:\"" + test_file( "/ugrid/1dtest/dflow1d_map.nc" ) + "\":" + "network" + ";;"
```

Comment:
I am still thinking if we shouldn't exclude "network". it looks like it is never real mesh so I would just not list it from UGRID driver at all. if someone complains, we can add it back...